docs/issue_guidance.md

```diff
@@ -0,0 +1,63 @@
+# Issue guidance
+
+This document details how Service Fabric handles issues reported through GitHub.
+
+**Note**, for support, guidance on opening issues, and general questions related to using Service Fabric, take a look
+at the documentation in the [contributing guide](/CONTRIBUTING.md).
+
+# Triaging issues
+
+Issues are triaged using GitHub labels, the following labels are used to indicate the status
+of the issue.
+
+## Bug, enhancement and question
+
+Most labels are used according the GitHub standards, these labels indicate the type of issue.
+
+|     Label     |                                             Description                                             |
+| ------------- | --------------------------------------------------------------------------------------------------- |
+| investigating | The issue isn't clear, further investigation is needed to determine if this a bug or something else |
+| bug           | The issue is related to a code or configuration defect in Service Fabric                            |
+| enhancement   | The issue is a request to improve a certain feature of Service Fabric                               |
+| question      | The issue is related to a clarification on how Service Fabric works                                 |
+
+
+## Area and needs triage
+
+Area labels and triage labels are used to identify the related involved components of Service Fabric.
+
+|    Label     |                                Description                                |
+| ------------ | ------------------------------------------------------------------------- |
+| needs triage | The involved component isn't clear, further investigation is needed       |
```

Comment:
do we want to introduce spaces in our tags? Maybe `needs-triage` instead? What do you think about sticking to dashes for multi-word tags? Whatever we choose we should be consistent, and maybe document somewhere on the GitHub the standard we've chosen.